### PR TITLE
store,server: Reduce TiFlashFallBack test time

### LINKED ISSUE #45773: TestTiFlashFallback took too long
## Bug Report

Please answer these questions before submitting your issue. Thanks!

### 3. What did you see instead (Required)

```
--- PASS: TestTiFlashFallback (211.23s)
```

### PR DESCRIPTION
**Pull request by yibin87**
### What problem does this PR solve?

Issue Number: close #45773 

Problem Summary:

### Check List

Tests 

- [x] Unit test
- [ ] Integration test
- [ ] Manual test (add detailed scripts or steps below)
- [ ] No code

Side effects

- [ ] Performance regression: Consumes more CPU
- [ ] Performance regression: Consumes more Memory
- [ ] Breaking backward compatibility

Documentation

- [ ] Affects user behaviors
- [ ] Contains syntax changes
- [ ] Contains variable changes
- [ ] Contains experimental features
- [ ] Changes MySQL compatibility

### Release note

Please refer to [Release Notes Language Style Guide](https://pingcap.github.io/tidb-dev-guide/contribute-to-tidb/release-notes-style-guide.html) to write a quality release note.

```release-note
None
```

### CODE REVIEW

File: store/copr/coprocessor.go [hunk 1]
```diff
@@ -1087,7 +1087,13 @@ func chooseBackoffer(ctx context.Context, backoffermap map[uint64]*Backoffer, ta
 	if ok {
 		return bo
 	}
-	newbo := backoff.NewBackofferWithVars(ctx, CopNextMaxBackoff, worker.vars)
+	boMaxSleep := CopNextMaxBackoff
+	if value, _err_ := failpoint.Eval(_curpkg_("ReduceCopNextMaxBackoff")); _err_ == nil {
```
[the remark this answers is not in the document]
Comment (yibin87) [reply store/copr/coprocessor.go]: Done

**Review by hawkingrei — APPROVED** [review]

**Review by Defined2014 — APPROVED** [review]

**Pull request merged**